### PR TITLE
fix(medusa-plugin-economic,medusa-plugin-mailchimp,medusa-plugin-restock-notification,medusa-plugin-sendgrid,medusa-plugin-wishlist): Temporarily remove payload validation in some plugins

### PR DESCRIPTION
**Pull request by StephixOne**
**What**

Commented out the payload validation bits in some plugins. Chose to comment-out instead of remove to keep the shape on hand for re-implementing validation with a new method.

**Why**

These plugins were attempting to validate the request body using a removed validator and crashing. After some back-and-forth in Slack, we landed on the solution above, pending updating the plugins to typescript and making use of our current validation strategies.

### CODE REVIEW

File: packages/medusa-plugin-wishlist/src/api/store/customers.js [hunk 1]
```diff
@@ -1,107 +1,75 @@
...
 import jwt from "jsonwebtoken"
+import { MedusaError } from "medusa-core-utils"
 
 const JWT_SECRET = process.env.JWT_SECRET || ""
 
 export default () => {
   const app = Router()
 
   app.delete("/:id/wishlist", bodyParser.json(), async (req, res) => {
-    const schema = Validator.object().keys({
```
Comment (adrien2p) [inline packages/medusa-plugin-wishlist/src/api/store/customers.js:12]: todo: add wrapMiddleware to handle the handle any error through the error-handler middleware globally applied

Comment (adrien2p) [reply packages/medusa-plugin-wishlist/src/api/store/customers.js:12]: same for all end points

File: packages/medusa-plugin-wishlist/src/api/store/customers.js [hunk 2]
```diff
@@ -1,107 +1,75 @@
...
+    })
 
-      res.json({ customer })
-    } catch (err) {
-      throw err
-    }
+    res.json({ customer })
   })
 
   app.post("/:id/wishlist", bodyParser.json(), async (req, res) => {
```
Comment (adrien2p) [inline packages/medusa-plugin-wishlist/src/api/store/customers.js:27]: suggestion: maybe move bodyParser at the top in app.use

**Review by adrien2p — APPROVED** [review]
I ve done some comments, but since those plugins need to be re written feel free to discard

**Comment by olivermrbl** [comment]
I will leave out the `medusa-plugin-discount-generation` as it needs to be deprecated. The implementation is from when we used MongoDB 😳

**Pull request merged**